src/app/shared/components/submission-state/submission-state.component.ts

```diff
@@ -19,38 +25,33 @@ export class SubmissionStateComponent {
    * @returns {string}
    */
   getStateChipClassName(submissionState: string): string {
-
-    if (submissionState === 'Pending' || submissionState === 'Draft') {
-      return 'warning';
-    }
-
-    if (submissionState === 'Valid') {
-      return 'success';
-    }
-
-    if (submissionState === 'Validating') {
-      return 'info';
-    }
-
-    if (submissionState === 'Invalid') {
-      return 'danger';
-    }
-
-    if (submissionState === 'Submitted') {
-      return 'secondary';
-    }
-
-    if (submissionState === 'Processing'
-      || submissionState === 'Cleanup'
-      || submissionState === 'Archiving'
-      || submissionState === 'Exporting') {
-      return 'warning-invert';
-    }
-
-    if (submissionState === 'Complete' || submissionState === 'Exported' || submissionState === 'Archived') {
-      return 'success-invert';
+    switch(submissionState) {
```

Comment:
Is there a reason not to use a map?

Reply:
I think this is easier to reason about than a map